src/ui/colortableview.js

```diff
@@ -252,6 +240,38 @@ export default class ColorTableView extends View {
 		this.keystrokes.listenTo( this.element );
 	}
 
+	/**
+	 * Renders and appends {@link #staticColorsGrid} and {@link #documentColorsGrid} views.
```

Comment:
SRP – render or append. Or change the name at least.

Reply:
The thing here is that in order to append it it also has to render it first (and this is the most intense operation, which we want to avoid). I'll turn it into append then.